frontend/src/Views/Export/ExportPage.tsx

```diff
@@ -1,14 +1,46 @@
 /**
- * Hello World page. Nothing Special. This is a skeleton!!!
- * @desc only shows Hello World, will implement as Designer creates mockup
- * @author @Kcaparas
+ * @desc Export page for downloading your work in nmp format
+ * @author @GDamaso
  */
 
-import React from 'react';
+import Button from '@Commons/Button/Button.tsx';
+import MainPageHeader from '@Commons/MainPageHeader/MainPageHeader.tsx';
+import MainPageFooter from '@Commons/MainPageFooter/MainPageFooter.tsx';
+import { FC } from 'react';
+import { StyledContent, StyledLandingContainer } from './ExportPage.styles.ts';
 
-const ExportPage: React.FC = () => {
-  localStorage.clear();
-  return <h2>Export Page</h2>;
+const ExportPage: FC = () => {
+  const downloadFile = () => {
+    localStorage.clear();
+    console.log('Downloading');
+  };
+
+  return (
+    <StyledLandingContainer>
+      <MainPageHeader />
+      <StyledContent>
+        <div id="dataFileHeader">
+          <h6>NMP Data File </h6>
+        </div>
+        <p>To continue later, Download this file to your computer or mobile device.</p>
+        <p>Load a NMP file on the Homepage when you want to continue.</p>
+        <a href="https://nmp.apps.nrs.gov.bc.ca/Report/Report#">
+          <p>How to use this data file</p>
+        </a>
+        <Button
+          size="md"
+          text="Download file"
+          handleClick={downloadFile}
+        />
+      </StyledContent>
+      <Button
+        size="lg"
+        text="Return to Calculation"
```

Comment:
I assume this is because of it being a stub file, but if I press `Return to Calculation` I lost all my progress in the forms